addons/mail/wizard/invite.py

```diff
@@ -47,7 +48,9 @@ def default_get(self, fields):
     send_mail = fields.Boolean('Send Email', default=True, help="If checked, the partners will receive an email warning they have been added in the document's followers.")
 
     def add_followers(self):
-        email_from = self.env['mail.message']._get_default_from()
+        if not self.env.user.partner_id.email:
+            raise UserError(_("Unable to post message, please configure the sender's email address."))
+        email_from = self.env.user.email_formatted
```

Comment:
I guess `user.partner_id.email` and `user.email_formatted` are tightly related.
But it feels weird to test one and then use the other.

Reply:
They are the same actually as email and email_formatted come from the partner (inherits). I can simplify code indeed.